css/index.css

```diff
@@ -0,0 +1,352 @@
+
+body {
+  font-family: Ubuntu,sans-serif;
+}
+
+.pb-21 {
+  background-color: white;
+  padding-bottom: 21px;
+  border-bottom: 1px solid #cbcbcb;
+}
+
+#form-color {
```

Comment:
@indalgit use of id for css isn't a good idea... Do read about **why to use class for css instead of id** ?